stock_delivery_slip_aggregate_quantity/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_aggregated_move_quantity,aggregated_move_quantity,stock_delivery_slip_aggregate_quantity.model_aggregated_move_quantity,base.group_user,1,1,1,1
```

Comment:
Please double-check if this should be how we define the ACL for this model, considering that the report may be downloaded by portal users.